### PR TITLE
fix: Wrong CSS globals

### LINKED ISSUE #1831: The ListCard options menu is shifted to the left

### LINKED ISSUE #1835: Modal buttons don't have margins between them

### PR DESCRIPTION
**Pull request by LautaroPetaccio**
This PR fixes some UI styles that were overwritten by different css files.
These files were using the `:global` directive or a `.css` file without scoping the directives to a component, making them to be applied in all of the site.

Closes #1831
Closes #1835

### CODE REVIEW

File: webapp/src/components/AssetPage/BaseDetail/BaseDetail.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ const BaseDetail = ({
             )}
           </div>
         </div>
-        {below}
+        <div className={'below'}>{below}</div>
```
Comment (kevinszuchet) [inline webapp/src/components/AssetPage/BaseDetail/BaseDetail.tsx:51]: Doesn't the `display: block` of the div break the component's styles? If it does, we could use a `span` instead.

File: webapp/src/components/AssetTopbar/SelectedFilters/Pill/Pill.tsx [hunk 2]
```diff
@@ -1,17 +1,31 @@
 import { Icon } from 'decentraland-ui'
+import classnames from 'classnames'
```
Comment (kevinszuchet) [inline webapp/src/components/AssetTopbar/SelectedFilters/Pill/Pill.tsx:2]:
```suggestion
import classNames from 'classnames'
```

File: webapp/src/components/AssetTopbar/SelectedFilters/Pill/Pill.tsx [hunk 3]
```diff
@@ -1,17 +1,31 @@
...
+export const Pill = ({
+  id,
+  label,
+  className,
+  onDelete
+}: Props): JSX.Element => {
   return (
-    <div className={styles.pill} data-testid={`pill-${id}`}>
+    <div
+      className={classnames(styles.pill, className)}
```
Comment (kevinszuchet) [inline webapp/src/components/AssetTopbar/SelectedFilters/Pill/Pill.tsx:20]:
```suggestion
      className={classNames(styles.pill, className)}
```

File: webapp/src/components/Modals/AssetFiltersModal/AssetFiltersModal.module.css [hunk 4]
```diff
@@ -58,6 +58,7 @@
   margin-bottom: 0;
 }
 
+/* This MUST be changed in the UI as this removes the padding from all modals */
```
Comment (kevinszuchet) [inline webapp/src/components/Modals/AssetFiltersModal/AssetFiltersModal.module.css:61]: Is it not possible to use a specific class name for this modal?

File: webapp/src/components/Modals/CreateOrEditListModal/CreateOrEditListModal.module.css [hunk 5]
```diff
@@ -12,6 +12,17 @@
   gap: 15px;
 }
 
+.checkbox:global(.ui.checkbox input[type='checkbox'].hidden ~ label:before),
+.checkbox:global(.ui.checkbox
+    input[type='checkbox'].hidden
+    ~ label:hover:before) {
+  border: 1px solid white;
```
Comment (flobarreto) [inline webapp/src/components/Modals/CreateOrEditListModal/CreateOrEditListModal.module.css:19]: isn't this solved by the ui update?

File: webapp/src/components/Modals/AssetFiltersModal/AssetFiltersModal.module.css (hunk 4 above)
Comment (LautaroPetaccio) [reply webapp/src/components/Modals/AssetFiltersModal/AssetFiltersModal.module.css:61]: Not by using the `className` property in the Modal component as this is pointing to the Modal's wrapper.

File: webapp/src/components/AssetTopbar/SelectedFilters/Pill/Pill.tsx (hunk 3 above)
Comment (LautaroPetaccio) [reply webapp/src/components/AssetTopbar/SelectedFilters/Pill/Pill.tsx]: Changed it!

File: webapp/src/components/AssetTopbar/SelectedFilters/Pill/Pill.tsx (hunk 2 above)
Comment (LautaroPetaccio) [reply webapp/src/components/AssetTopbar/SelectedFilters/Pill/Pill.tsx]: Changed it!

File: webapp/src/components/AssetPage/BaseDetail/BaseDetail.tsx (hunk 1 above)
Comment (LautaroPetaccio) [reply webapp/src/components/AssetPage/BaseDetail/BaseDetail.tsx]: Changed it!

**Review by kevinszuchet — APPROVED** [review]

**Pull request merged**